src/tools/rbd/action/DiskUsage.cc

```diff
@@ -286,7 +317,9 @@ void get_arguments(po::options_description *positional,
   options->add_options()
     (at::FROM_SNAPSHOT_NAME.c_str(), po::value<std::string>(),
      "snapshot starting point")
-    ("exact", po::bool_switch(), "compute exact disk usage (slow)");
+    ("exact", po::bool_switch(), "compute exact disk usage (slow)")
+    ("merge-snapshots", po::bool_switch(),
```

Comment:
Looks like it's already an oversight, but can you add `Shell::SwitchArguments switched_arguments({"exact", "merge-snapshots"});` down near the `Shell::Action` definitions?

Reply:
@illaman Sure, here we go
However, I do not understand what this code do, can you explain it to me ?

Reply:
It helps the argument validator in `Shell.cc` to know which optionals do not have values as the next parameter. Otherwise, you could run "rbd du --merge-snapshots=blah ..."